<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KWInsertInlineNoteDialog</class>
 <widget class="QWidget" name="KWInsertInlineNoteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>209</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Numbering</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="automaticNumbering">
        <property name="text">
         <string>Automatic</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="characterNumbering">
          <property name="text">
           <string>Character</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="characterEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="footnoteType">
        <property name="text">
         <string>Footnote</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="endnoteType">
        <property name="text">
         <string>Endnote</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
